swap/peer.go

```diff
@@ -74,15 +74,10 @@ func (sp *Peer) handleMsg(ctx context.Context, msg interface{}) error {
 // a cheque from a creditor
```

Comment:
handleEmitChequeMsg should be handled by the *creditor* when it receives
a cheque from a *creditor*. This sounds wrong to me. I think it should be:
handleEmitChequeMsg should be handled by the *creditor* when it receives
a cheque from a *debitor*